packages/angular/test/tracing.test.ts

```diff
@@ -137,81 +235,209 @@ describe('Angular Tracing', () => {
             },
           },
           '/org/:orgId/projects/:projId/events/:eventId/',
+          [
+            {
+              path: 'org/:orgId',
+              component: AppComponent,
+              children: [
+                {
+                  path: 'projects/:projId',
+                  component: AppComponent,
+                  children: [
+                    {
+                      path: 'events/:eventId',
+                      component: AppComponent,
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
         ],
-      ])('%s and sets the source to `route`', (_, url, routerState, result) => {
-        instrumentAngularRouting(customStartTransaction, false, true);
+      ])('%s and sets the source to `route`', async (_, url, routerState, result, routes) => {
+        const customStartTransaction = jest.fn(defaultStartTransaction);
+        const env = await TestEnv.setup({
+          customStartTransaction,
+          routes,
+          startTransactionOnPageLoad: false,
+        });
 
-        // this event starts the transaction
-        routerEvents$.next(new NavigationStart(0, url));
+        await env.navigateInAngular(url, routerState);
```

Comment:
I might be missing something but why are we passing in the `routerState` here? Since we now have the `TestBed` and we can perform actual navigations, shouldn't the router internally build up the router state based on the passed `routes` and the `url`?

Reply:
Yes, turns out it's not needed anymore. Updated the tests and the utility function. 👍

Reply:
Awesome, thanks 👍